### PR TITLE
fixed issue with search notes. Changed the action type to match what its intended.

### PR DESCRIPTION
**Pull request by leelakrishnan**

### CODE REVIEW

File: lib/Model/NLUAction.dart [hunk 1]
```diff
@@ -2,8 +2,9 @@ import 'package:json_annotation/json_annotation.dart';
 
 @JsonSerializable(nullable: true)
```
Comment (prestech) [inline lib/Model/NLUAction.dart]: Got the following error while trying to build this is the following command `flutter pub run build_runner build --delete-conflicting-outputs`

`The class `ActionType` has no default constructor.
package:untitled3/Model/NLUAction.dart:4:6`


The error disappeared when I commented out the @JsonSerializable(nullable: true)

Comment (prestech) [reply lib/Model/NLUAction.dart]: @JsonSerializable(nullable: true) expects every Class to define a default constructor (but this ENUM does not have one).

Comment (leelakrishnan) [reply lib/Model/NLUAction.dart]: Fixed the issue The class ActionType has no default constructor

**Review by ahrohn92 — APPROVED** [review]
Looks good.

**Pull request merged**